### DISCUSSION TITLE
How to hide certain dotfiles like .gitignore .DS_Store etc. but not others

### DISCUSSION THREAD

**Discussion by GeordyJ** [gokcehan/lf#1585] [Q&A] [answered]
I usually have 'set hidden' in my lfrc as I frequently edit dot files. However, I do not want to see files like .DS_Store and files that start with ._ 

Thanks

**Comment by joelim-work** [comment, marked as answer]
Instead of using `set hidden` to forcefully show hidden files, you can configure the [`hiddenfiles`](https://github.com/gokcehan/lf/blob/master/doc.md#hiddenfiles-string-default-) option to define what should be considered as a hidden file.

The following will treat `.DS_Store` and files starting with `._` as hidden:

```
set hidden false  # this is the default already

set hiddenfiles '.DS_Store:._*'
```

**Reply by GeordyJ** [reply]
Thank you!
